Doppler.Sap/Mappers/Dictionaries.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Doppler.Sap.Mappers
+{
+    public static class Dictionaries
```

Comment:
Exist a [Dictionary.cs](https://github.com/FromDoppler/doppler-sap/blob/d2f139e20c37ceb0b2ef8614e96915ac3b4049a5/Doppler.Sap/Mappers/Dictionary.cs) class, we can have one dictionary with all dictionaries, maybe we can remove the Dictionary class and move the content in this class

Reply:
You're right, my bad, I didnt pay attention